x11-misc/xscreensaver/xscreensaver-6.05-r2.ebuild

```diff
@@ -0,0 +1,248 @@
+	"${FILESDIR}"/${PN}-6.01-without-gl-makefile.patch
+	"${FILESDIR}"/${PN}-6.01-non-gtk-install.patch
+	"${FILESDIR}"/${PN}-6.01-configure-install_sh.patch
+	"${FILESDIR}"/${PN}-6.03-without-gl-configure.patch
+	"${FILESDIR}"/${PN}-6.05-remove-update-icon-cache.patch
+	"${FILESDIR}"/${PN}-6.05-r2-configure-exit-codes.patch
+	"${FILESDIR}"/${PN}-6.05-get-dirs-from-gtk3.0-in-configure.patch
+)
+
+DOCS=( README{,.hacking} )
+
+src_prepare() {
+	default
+
+	sed -i configure.ac -e '/^ALL_LINGUAS=/d' || die
+	strip-linguas -i po/
+	export ALL_LINGUAS="${LINGUAS}"
+
+	if use new-login && ! use gdm; then #392967
+		sed -i \
+			-e "/default_l.*1/s:gdmflexiserver -ls:${EPREFIX}/usr/libexec/lightdm/&:" \
+			configure{,.ac} || die
+	fi
+
+	# We are patching driver/XScreenSaver.ad.in, so let's delete the
+	# header generated from it so that it gets back in sync during build:
+	rm driver/XScreenSaver_ad.h || die
+
+	if ! use offensive; then
+		sed -i \
+			-e '/boobies/d;/boobs/d;/cock/d;/pussy/d;/viagra/d;/vibrator/d' \
+			hacks/barcode.c || die
+		sed -i \
+			-e 's|erect penis|shuffle board|g' \
+			-e 's|flaccid penis|flaccid anchor|g' \
+			-e 's|vagina|engagement ring|g' \
+			-e 's|Penis|Shuttle|g' \
+			hacks/glx/glsnake.c || die
+		sed -i \
+			's| Stay.*fucking mask\.$||' \
+			hacks/glx/covid19.man \
+			hacks/config/covid19.xml || die
+		eapply "${FILESDIR}/xscreensaver-6.05-teach-handsy-some-manners.patch"
+	fi
+
+	config_rpath_update "${S}"/config.rpath
+
+	# Must be eauto*re*conf, to force the rebuild
+	eautoreconf
+}
+
+src_configure() {
+	if use ppc || use ppc64; then
+		filter-flags -maltivec -mabi=altivec
+		append-flags -U__VEC__
+	fi
+
+	unset BC_ENV_ARGS #24568
+
+	# /proc/interrupts won't always have the keyboard bits needed
+	# Not clear this does anything in 6.03+(?) but let's keep it for now in case.
+	# (See also: configure argument)
+	export ac_cv_have_proc_interrupts=yes
+
+	# WARNING: This is NOT a normal autoconf script
+	# Some of the --with options are NOT standard, and expect "--with-X=no" rather than "--without-X"
+	ECONF_OPTS=(
+		$(use_enable locking)
+		$(use_with elogind)
+		$(use_with gdk-pixbuf pixbuf)
+		$(use_with gles)
+		$(use_with glx)
+		$(use_with gtk)
+		$(use_with new-login login-manager)
+		$(use_with pam)
+		$(use_with suid setuid-hacks)
+		$(use_with systemd)
+		$(use_with xinerama xinerama-ext)
+		--with-jpeg=$(usex jpeg yes no)
+		--with-png=$(usex png yes no)
+		--with-xft=yes
+		--with-app-defaults="${EPREFIX}"/usr/share/X11/app-defaults
+		--with-configdir="${EPREFIX}"/usr/share/${PN}/config
+		--with-dpms-ext
+		--with-hackdir="${EPREFIX}"/usr/$(get_libdir)/misc/${PN}
+		--with-proc-interrupts
+		--with-randr-ext
+		--with-text-file="${EPREFIX}"/etc/gentoo-release
+		--with-xdbe-ext
+		--with-xf86gamma-ext
+		--with-xf86vmode-ext
+		--with-xinput-ext
+		--with-xkb-ext
+		--with-xshm-ext
+		--without-gle
+		--without-kerberos
+		--without-motif
+		--with-proc-oom=yes
```

Comment:
`-r2` seems to add `=yes` superflous `=yes` here. My vote for a revert, without it was cleaner. I checked both https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/External-Software.html and the `configure.ac` code.